pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2815,7 +2815,7 @@ func (c *Cloud) removeSecurityGroupIngress(securityGroupID string, removePermiss
 // Makes sure the security group exists.
 // For multi-cluster isolation, name must be globally unique, for example derived from the service UUID.
 // Returns the security group id or error
-func (c *Cloud) ensureSecurityGroup(name string, description string) (string, error) {
+func (c *Cloud) ensureSecurityGroup(name string, description string, annotations map[string]string) (string, error) {
```

Comment:
Nit: let's call them "tags" not "annotations"?  Maybe "extraTags" or "additionalTags" ?

It's would also be nice to apply the tags where we didn't previously create them.  I think there's an `additionalTags` argument to readRepairClusterTags which will do the right thing.  If it's complicated we can address this later, but I _think_ it's simple (?)